src/scheduler/data_types.h

```diff
@@ -854,9 +854,6 @@ struct resresv_set
 	place *place_spec;		/* place spec of set */
 	resource_req *req;		/* ATTR_L (qsub -l) resources of set.  Only contains resources on the resources line */
 	queue_info *qinfo;		/* The queue the resresv is in if the queue has nodes associated */
-
-	resource_resv **resresv_arr;	/* The resresvs in the set */
```

Comment:
did you remove this because it was not used?

Reply:
Yes, resresv_arr stored the list of all jobs in the equivalence class but we were not using it anywhere. Thus removed it.

Reply:
Yeah, this was left over when I did the original POC of equivalence classes.  In the original POC, when a job in a class could not run, we'd mark all jobs in the class at that point.  I'd walk down the resresv_arr then.  This disturbed the political order.  When the final equivalence class RFE was finished, we now mark the class as can not run and when we reach new jobs from next_job(), we immediately know they can't run.  This way the political order is not changed.